client-tools/Benchmark/BenchFeature.cpp

```diff
@@ -250,6 +257,18 @@ void BenchFeature::validateOptions(std::shared_ptr<ProgramOptions> options) {
           << _percentiles << "': histogram is disabled by default. Enable it with flag '--histogram.generate = true'.";
     }
   }
+  LOG_DEVEL << _customQuery;
+  if (!_customQueryBindVars.empty()) {
+    _customQueryBindVarsBuilder = VPackParser::fromJson(_customQueryBindVars);
```

Comment:
What happens if the specified JSON is invalid? Will the end user get a proper error message in that case?

Reply:
Addressed